src/fiatconnect/KycLanding.tsx

```diff
@@ -121,6 +131,13 @@ export function KycAgreement(props: { personaKycStatus?: KycStatus; quote: FiatC
   }
 
   const personaSuccessCallback = () => {
+    // optimistally navigate to step 2 to reduce flash
+    // The subsequent saga may direct the user elsewhere if there is an error or edge case
```

Comment:
I guess we could also let the user continue setting up the bank again and retry in the background, since the failure is likely transient, or just show an error screen later, but we could add those later as well.